machine_learning_hep/do_entire_analysis.py

```diff
@@ -1,5 +1,5 @@
 #############################################################################
-##  © Copyright CERN 2018. All rights not expressly granted are reserved.  ##
+##  © Copyright CERN 2018. All rights not expressly granted are reserved.  ## 
```

Comment:
We should not change the copyright. That makes the continuous integration fail

Reply:
My bad ;)